## 11_fix_dbpoolx_mod_header_comment_pubid.dpatch by Daniel Leidert (dale) <daniel.leidert@wgdd.de>.
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: There are some wrong references to the PUBID 
## DP: "-//OASIS//ELEMENTS DocBook XML Information Pool V4.x//EN", which should be
## DP: "-//OASIS//ELEMENTS DocBook Information Pool V4.x//EN". This patch fixes them.
## DP:
## DP: <URL:https://sourceforge.net/tracker/?func=detail&atid=384107&aid=1679712&group_id=21935>
## DP:
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.5-1>
## DP:
## DP: # only apply to DB >= 4.2!
## DP: for i in `find . -type f` ; do \
## DP: 		sed -e 's/ELEMENTS DocBook XML Information Pool/ELEMENTS DocBook Information Pool/g' < $i > $i.new ; \
## DP: 		mv $i.new $i ; \
## DP: done

diff -urNad docbook-xml-4.5~/docbook-4.2/dbpoolx.mod docbook-xml-4.5/docbook-4.2/dbpoolx.mod
--- docbook-xml-4.5~/docbook-4.2/dbpoolx.mod	2002-07-17 18:14:41.000000000 +0200
+++ docbook-xml-4.5/docbook-4.2/dbpoolx.mod	2007-03-25 20:10:29.000000000 +0200
@@ -50,7 +50,7 @@
      declaration that uses the public identifier shown below:
 
      <!ENTITY % dbpool PUBLIC
-     "-//OASIS//ELEMENTS DocBook XML Information Pool V4.2//EN"
+     "-//OASIS//ELEMENTS DocBook Information Pool V4.2//EN"
      "dbpoolx.mod">
      %dbpool;
 
diff -urNad docbook-xml-4.5~/docbook-4.3/dbpoolx.mod docbook-xml-4.5/docbook-4.3/dbpoolx.mod
--- docbook-xml-4.5~/docbook-4.3/dbpoolx.mod	2004-03-31 13:20:20.000000000 +0200
+++ docbook-xml-4.5/docbook-4.3/dbpoolx.mod	2007-03-25 20:10:29.000000000 +0200
@@ -50,7 +50,7 @@
      declaration that uses the public identifier shown below:
 
      <!ENTITY % dbpool PUBLIC
-     "-//OASIS//ELEMENTS DocBook XML Information Pool V4.3//EN"
+     "-//OASIS//ELEMENTS DocBook Information Pool V4.3//EN"
      "dbpoolx.mod">
      %dbpool;
 
diff -urNad docbook-xml-4.5~/docbook-4.4/dbpoolx.mod docbook-xml-4.5/docbook-4.4/dbpoolx.mod
--- docbook-xml-4.5~/docbook-4.4/dbpoolx.mod	2005-01-27 14:57:19.000000000 +0100
+++ docbook-xml-4.5/docbook-4.4/dbpoolx.mod	2007-03-25 20:10:29.000000000 +0200
@@ -50,7 +50,7 @@
      declaration that uses the public identifier shown below:
 
      <!ENTITY % dbpool PUBLIC
-     "-//OASIS//ELEMENTS DocBook XML Information Pool V4.4//EN"
+     "-//OASIS//ELEMENTS DocBook Information Pool V4.4//EN"
      "dbpoolx.mod">
      %dbpool;
 
diff -urNad docbook-xml-4.5~/docbook-4.5/dbpoolx.mod docbook-xml-4.5/docbook-4.5/dbpoolx.mod
--- docbook-xml-4.5~/docbook-4.5/dbpoolx.mod	2006-10-03 15:26:00.000000000 +0200
+++ docbook-xml-4.5/docbook-4.5/dbpoolx.mod	2007-03-25 20:10:29.000000000 +0200
@@ -50,7 +50,7 @@
      declaration that uses the public identifier shown below:
 
      <!ENTITY % dbpool PUBLIC
-     "-//OASIS//ELEMENTS DocBook XML Information Pool V4.5//EN"
+     "-//OASIS//ELEMENTS DocBook Information Pool V4.5//EN"
      "dbpoolx.mod">
      %dbpool;
 
